packages/ia-topnav/src/data/menus.js

```diff
@@ -1,4 +1,6 @@
-const texts = baseUrl => ({
+import { prodUrl } from '../constants';
+
+const texts = (baseUrl = prodUrl) => ({
```

Comment:
SUGGESTION: Here and throughout, again wondering if `baseHost` or `baseHostname` would be more accurate than `baseUrl`.